settings.gradle

```diff
@@ -1 +1,20 @@
-include ':AutomatticTracks'
+pluginManagement {
```

Comment:
[Plugins DSL](https://docs.gradle.org/current/userguide/plugins.html#sec:plugins_block) is the favored way of declaring Gradle plugins, in opposite to `apply plugin:''`

Here I declare versions of `Android Gradle Plugin` and `Kotlin Android` plugin for both modules: `lib` and `sampletracksapp`